generator/src/test/java/com/scottlogic/deg/generator/cucumber/OfType.feature

```diff
@@ -3,201 +3,94 @@ Feature: User can specify that a field is of a specific type (string, numeric or
 Background:
      Given the generation strategy is full
 
-Scenario: Running an 'ofType' = String request that includes roman character strings that include in-use values should be successful
-     Given there is a field foo
-       And foo is in set:
-       | "true"      |
-       | "false"     |
-       | "null"      |
-       | "undefined" |
-       And foo is of type "string"
-     Then the following data should be generated:
-       | foo         |
-       | null        |
-       | "true"      |
-       | "false"     |
-       | "null"      |
-       | "undefined" |
-
-Scenario: Running an 'ofType' = String request that includes strings with special characters should be successful
-     Given there is a field foo
-       And foo is in set:
-       | "!?:;()&%+-="                                |
-       | "]	[] [] [] [] [] [] ["                       |
-       | "†ŠŒŽ™¼ǅ©®…¶Σ֎"                               |
-       | "☺☹☻😀😁😂😃😄😅😆😇😈😉😊😋😌🚩🚪🚫🚬🚭🚮🚯🚰" |
-       | "传/傳象形字ФХѰѾЦИتشرقصف"                      |
-       And foo is of type "string"
-     Then the following data should be generated:
-       | foo                                            |
-       | null                                           |
-       | "!?:;()&%+-="                                  |
-       | "]	[] [] [] [] [] [] ["                       |
-       | "†ŠŒŽ™¼ǅ©®…¶Σ֎"                               |
-       | "☺☹☻😀😁😂😃😄😅😆😇😈😉😊😋😌🚩🚪🚫🚬🚭🚮🚯🚰" |
-       | "传/傳象形字ФХѰѾЦИتشرقصف"                      |
-
-Scenario: Running an 'ofType' = String request that includes strings with special characters (standard) alongside roman alphanumeric characters should be successful
-     Given there is a field foo
-       And foo is in set:
-       | "abcdefghijklmnop  !?:;()&%+-="                                  |
-       | "abcdefghijklmnop  ]	[] [] [] [] [] [] ["                       |
-       | "abcdefghijklmnop  †ŠŒŽ™¼ǅ©®…¶Σ֎"                               |
-       | "abcdefghijklmnop  ☺☹☻😀😁😂😃😄😅😆😇😈😉😊😋😌🚩🚪🚫🚬🚭🚮🚯🚰" |
```

Comment:
How cmoe the emoji test has been removed?

Reply:
😢